### DISCUSSION TITLE
Refresh and save automatically at intervals

### DISCUSSION THREAD

**Discussion by freemedom** [gildas-lormeau/SingleFile#1141] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Is there no feature to open/**refresh** a url and save it every once in a while (for example, a hour)? I want to periodically save a url that updates frequently.
(It doesn't matter if this don't have this feature, I can do this with another browser extension that regularly refreshes pages in conjunction with SingleFile's "Auto-save". So that other people with similar needs can search to the issue.)
Thank you!

**Describe alternatives you've considered (optional)**

**Additional context (optional)**

**Comment by gildas-lormeau** [comment]
I confirm that auto-refreshing the page is out of the scope of the extension today. However, as you have noticed, a third party extension + auto save in SingleFile fixes this issue.
